### PR TITLE
numatune: Fix problems and update get numa parameter method

### PR DESCRIPTION
**Pull request by waynesun09**
Signed-off-by: Wayne Sun <gsun@redhat.com>

### CODE REVIEW

**Comment by waynesun09** [comment]
This depends on
https://github.com/autotest/virt-test/pull/1999

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_numatune.cfg [hunk 1]
```diff
@@ -127,6 +128,7 @@
                                         - interleave:
                                             numa_mode = 'interleave'
                                         - strict:
+                                            numa_nodeset = 0
```
Comment (yumingfei) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_numatune.cfg]: I have one question that whether parameters in test module should be changed in same format(Integer)?
Such as following code:
```
nodeset = params.get("numa_nodeset", "")
```

Comment (waynesun09) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_numatune.cfg]: @yumingfei as I remember the parameters parsed from cfg file are string type, so the integer you see in cfg will be string value in params dict.
I just tested, it did is string value.
As whether to change to same format, it depends on the how it used in the module, nodeset in virsh numatune command is string format.

Comment (yumingfei) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_numatune.cfg]: OK, it's acceptable.

**Pull request merged**